src/logging/timer.h

```diff
@@ -93,12 +93,10 @@ class Timer
 } // namespace BCLog
 
 
-#define LOG_TIME_MICROS(end_msg, ...) \
-    BCLog::Timer<std::chrono::microseconds> PASTE2(logging_timer, __COUNTER__)(__func__, end_msg, ## __VA_ARGS__)
-#define LOG_TIME_MILLIS(end_msg, ...) \
-    BCLog::Timer<std::chrono::milliseconds> PASTE2(logging_timer, __COUNTER__)(__func__, end_msg, ## __VA_ARGS__)
-#define LOG_TIME_SECONDS(end_msg, ...) \
-    BCLog::Timer<std::chrono::seconds> PASTE2(logging_timer, __COUNTER__)(__func__, end_msg, ## __VA_ARGS__)
+#define LOG_TIME_MILLIS(end_msg, log_category) \
+    BCLog::Timer<std::chrono::milliseconds> PASTE2(logging_timer, __COUNTER__)(__func__, end_msg, log_category)
+#define LOG_TIME_SECONDS(end_msg) \
+    BCLog::Timer<std::chrono::seconds> PASTE2(logging_timer, __COUNTER__)(__func__, end_msg)
```

Comment:
So while the macro name indicates the only difference is the precision, they are functionally also different to the logging behaviour: One always logs, the other must accept a category. Not sure if it makes sense to keep the names so similar then.

Reply:
Ok. I've renamed `LOG_TIME_MILLIS` to `LOG_TIME_MILLIS_WITH_CATEGORY`.